params.go

```diff
@@ -45,6 +46,18 @@ type StringSliceParam struct {
 	value []string
 }
 
+type URLParam struct {
+	baseParam
+	def   string
```

Comment:
Right, it parses it only in the config file, otherwise it wants to be passed something of type time.Duration.